src/main/java/com/forgeessentials/util/output/ChatOutputHandler.java

```diff
@@ -155,7 +155,7 @@ public static void broadcast(TextComponent message, boolean sendToDiscord)
 
         if (sendToDiscord && ModuleChat.instance != null)
         {
-            ModuleChat.instance.discordHandler.sendMessage(message.getString());
+            ModuleChat.instance.discordMessageHandler.sendMessage(message.getString());
```

Comment:
This could probally be simplified by just checking for the existence of the discord module (and instance) instead of piggybacking it on the chat module.